docs/contributing.rst

```diff
@@ -132,6 +132,29 @@ mean that this work should be relatively manageable.
 We hope that the lessons we learn building something useful in the short term will be
 applicable in the future as other needs arise.
 
+.. _backwards compatibility with meica:
+
+Is ``tedana`` backwards compatible with MEICA?
+``````````````````````````````````````````````
+
+The short answer is no.
+
+There are two main reasons why. One is that the tool originally used to run the independent
+component analysis core to the MEICA method (`mdp`_) is no longer supported. In November 2018
+the developers made the decision to switch to `scikit learn`_ to perform these analyses. Scikit
+learn is well supported and under long term development. ``tedana`` will be more stable and
+have better performance going forwards as a result of this switch, but it also means that
+exactly reproducing MEICA analyses is not possible.
+
+The other reason is a choice of the core developers to look forwards rather than maintaining
+an older codebase. As is described `above`_, ``tedana`` is developed by a small team of
```

Comment:
We could actually link this directly out with

```rst
As described in the ``tedana`` :ref:`governance` description,
```
I'm actually not sure where `above`_ is linking to right now ?

Reply:
Lolz. Yep. I think I did at some point have a link for "above" but it isn't there anymore!

I've changed this to:

`As described in the :ref:`governance` section, ``tedana`` is developed by a small...`

Is that the right formatting??

Reply:
It is ! but sorry, we need to add the link in-text. So above at [L11](https://github.com/emdupre/tedana/pull/18/files/32cbb260a94084a2e8f45c2a22b8d244fd0e4948#diff-33c6b4e83faefd142a5bcda932de6863L11) it should be:

```rst
.. _governance:

Governance
----------------
```

Reply:
DONE 👾